### DISCUSSION TITLE
Manipulate output

### DISCUSSION THREAD

**Discussion by Johnwulp** [jomjol/AI-on-the-edge-device#1750] [Q&A] [answered]
Hi,

Since my last digit is physically unreadable, its never detected correctly.
Is it possible to multiply the output value with 10 and omit the last digit? Or set the last digit to a fixed 0 value? 
I know that the results will be off, but at least the sensor in Homeassistant is correct without any modification.

<img width="654" alt="Screenshot" src="https://user-images.githubusercontent.com/879840/210281638-44aeacab-0f39-4c69-a1a9-bb4109451f49.png">

Thanks in advance!

**Comment by ghost** [comment, marked as answer]
You can configure the ROI smaller or you can use in decimal shift a positiv value.

**Reply by Johnwulp** [reply]
Thanks! I increased the decimalshift value to 1
